slices/libc6-dev.yaml

```
@@ -0,0 +1,120 @@
+package: libc6-dev
+
+essential:
+  - libc6-dev_copyright
+
+slices:
+  copyright:
+    contents:
+      /usr/share/doc/libc6-dev/copyright:
+
+  libs:
+    essential:
+      - libc6-dev_core-libs
+      - libc6-dev_headers
+      - libc6-dev_symlink-libc6-libs
+
+  core-libs:
+    contents:
+      # We have to list all the files since we're in a generic place (/usr/lib/*-linux-*)
+      # and if we tried to glob *.o files, this slice would conflict with oh-so-many
+
+      /usr/lib/*-linux-*/Mcrt1.o:
+      /usr/lib/*-linux-*/Scrt1.o:
+      /usr/lib/*-linux-*/crt1.o:
+      /usr/lib/*-linux-*/crti.o:
+      /usr/lib/*-linux-*/crtn.o:
+      /usr/lib/*-linux-*/gcrt1.o:
+      /usr/lib/*-linux-*/grcrt1.o:
+        arch: [ amd64, arm64, riscv64, s390x ]
```

```suggestion
        arch: [amd64, arm64, riscv64, s390x]
```

Let's make the formatting consistent across different files. We don't put extra spaces inside the brackets.